test/test_gpu/skip_tests_h100_pytorch.yaml

```diff
@@ -19,6 +19,7 @@ flash_attention:
   - triton_tutorial_flash_v2_ws
   - triton_tutorial_flash_v2_tma_ws
   - triton_tutorial_flash_v2_tma_ws_persistent
+  - triton_tutorial_flash_v2_bwd_ws
```

Comment:
Ah I see, yes, we need to include it in the skip list in this case.

Just asking, what does "bwd" mean here? Does it mean backwards?

Reply:
Yes it means backwards. I added two extra arguments to specify variants for fwd and bwd.

Reply:
I am wondering why we need to add explicit benchmark for backward? I thought it could be covered by calling `--mode bwd` and `get_bwd_fn()`: https://github.com/pytorch-labs/tritonbench/blob/main/tritonbench/operators/flash_attention/operator.py#L484

Reply:
Yeah, this diff does not add explicit benchmark for backward, we still use --bwd to run the backward kernel, but there are two implementations for the backward kernel and I used a different benchmark name to choose between the two variants. Open to other suggestions :]
Maybe we can add a command line arg, but I want to compare the accuracy between the two implementations.

Reply:
I see, then creating a separate benchmark sounds good to me.